src/h3/api/_api_template.py

```diff
@@ -61,9 +67,21 @@ def num_hexagons(resolution):
         return _cy.num_hexagons(resolution)
 
     def hex_area(resolution, unit='km^2'):
+        """ Return the average area of an H3 cell
+        for the given resolution.
+
+        todo: all cells, or just hexagons?
```

Comment:
This is for hexagonal cells only (see also https://github.com/uber/h3/issues/168)

Reply:
updating!